packages/dcos-launch/extra/launch/config.py

```diff
@@ -49,375 +21,256 @@ def expand_path(path: str, relative_dir: str) -> str:
     return os.path.abspath(os.path.join(relative_dir, path))
```

Comment:
Yeah, I noticed the test later.
And Yes, I agree, that it will be better to test the Validator class.